answer/negativeAnswer/error.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "http://pop2020.dedis.ch/answer/negativeAnswer/error.json",

```

Comment:
There was a thread on $id. Just came across this bit in the documentation which I had missed:
https://json-schema.org/understanding-json-schema/structuring.html#the-id-property
it refers to differences in library implementations and how id is used as a base path for relative references.

As such, @RaulinN had a point in suggesting:

> Could we maybe use the github URL? Maybe consider that all schemas are already on the proto-specs branch (thus "$id": "https://github.com/dedis/student20_pop/tree/proto-specs/<path-to-file>")

This would make most sense ! 
Sorry for coming back to you late on that one :)

Reply:
done!